layer/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+cd $(dirname $0)
+
+OS=Linux       # or Darwin, Windows
+ARCH=x86_64    # or arm64, x86_64, armv6, i386, s390x
+VERSION=v0.19.0
+
+curl -sL "https://github.com/google/go-containerregistry/releases/download/${VERSION}/go-containerregistry_${OS}_${ARCH}.tar.gz" > go-containerregistry.tar.gz
```

Comment:
Does this script run on customer's machine?

Reply:
Correct me if I'm wrong: this shell script will run whenever customers run `synth()`.

Instead of downloading it every time when customers use the construct, can we do this part in our release process and publish the `layer.zip`. So they don't need to connect to Github which has availability risk.